MacrobenchmarkSample/app/build.gradle

```diff
@@ -23,7 +23,7 @@ android {
     compileSdkVersion 31
     defaultConfig {
         applicationId "com.example.macrobenchmark.target"
-        minSdkVersion 14
+        minSdkVersion 21
```

Comment:
I raised it for simplified layout creation. Should I keep at 14 and hack around the layouts?

Reply:
Nah, 21 seems fine to me.